### ISSUE TITLE
Koala can't compile the material-dashboard.scss

### ISSUE THREAD

**Issue by aVolpe** [creativetimofficial/material-dashboard-pro-angular#2]
Following the tutorial in ./documentation/tutorial-components, the part:

# Changing Colors (SASS)

You can change the default colors via variables from SASS:

1. You can find the colors in assets/sass/material-dashboard/_variables.scss starting with line 28 where is the primary color set: $brand-primary: $purple !default;. All the possible colors are in assets/sass/material-dashboard/_colors.scss
1. Add the SASS folder from assets/sass/ to a new project inside Koala Compiler and find the file material-dashboard.scss, it will be the one in Green.
1. Right click on that file and set the output path, it must be in assets/css/ so anything that you compile will overwrite the original material-dashboard.css
1. Press on compile and everything will be done automatically.

When I press compile, koala throws this error:

Basically is the name of the scss file and nothing more, I see that the css file was changed, but I don't see the changes.

How can I solve this?

![screen shot 2017-04-06 at 6 34 31 pm](https://cloud.githubusercontent.com/assets/1097701/24780241/c66dff38-1b03-11e7-8c90-5f1d8ca4ee44.png)

Koala Version: latest, I downloaded it today 2.0.4
Material-Dashboard: latest, I buy and downloaded it today

**Comment by alexandru-paduraru** [comment]
@aVolpe thank you for using our product and sorry for the late response. I don't know why we missed this notification. Please deactivate the "Debug info" and "Source Map" and compile again and let us know your status.

Best,
Alex

**Comment by aVolpe** [comment]
Without "source map" works fine.
